packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.types.ts

```diff
@@ -86,6 +87,11 @@ export interface IBreadcrumbProps extends React.HTMLAttributes<HTMLElement> {
    * Focuszone props that will get passed through to the root focus zone.
    */
   focusZoneProps?: IFocusZoneProps;
+
+  /**
+   * Callout props that will get passed through to overflow tooltips.
+   */
+  calloutProps?: ICalloutProps;
```

Comment:
Can these be expanded to be any `TooltipHost` props instead of just for the callout? I can imagine wanting to do more customization of the tooltip, and it would be nice to enable that without further work. `focusZoneProps` above does this already and just passes all props through to the specific control.

Reply:
Ok sure, I can look into that! 👍